ipfx/script_utils.py

```diff
@@ -77,6 +77,14 @@ def dataset_for_specimen_id(specimen_id, data_source, ontology):
             logging.warning("Exception when loading specimen {:d} via Allen SDK".format(specimen_id))
             logging.warning(detail)
             return {"error": {"type": "dataset", "details": traceback.format_exc(limit=None)}}
+    elif data_source == "filesystem":
+        nwb_path = file_list[specimen_id]
+        try:
+            data_set = create_ephys_data_set(nwb_file=nwb_path)
```

Comment:
we should include `ontology` as an argument as well
On line 66 please remove `h5_file` we need to remove the argument from the `create_ephys_data_set`